atom/browser/api/lib/dialog.coffee

```diff
@@ -91,10 +91,11 @@ module.exports =
 
     throw new TypeError('Buttons need to be array') unless Array.isArray options.buttons
 
-    options.title ?= ''
-    options.message ?= ''
-    options.detail ?= ''
-    options.icon ?= null
+    options.title              ?= ''
+    options.message            ?= ''
+    options.detail             ?= ''
+    options.icon               ?= null
+    options.defaultId ?= -1
```

Comment:
The spacing in lines 94-98 is not consistent. There should be a single space before `?=`.